docs/webhooks/index.html

```diff
@@ -967,6 +972,51 @@ <h3 id="certificationRequestsMerged">certificationRequestsMerged</h3>
     deletedRequestId: 'lehjeh36dleh29',
     mergedRequestId: 'dhey29hcl28hl28',
   }
+};</code></pre>
+            </div>
+            <h2>Events for CodeSignal Live Interview</h2>
+            <div class="subsection">
+              <h3 id="liveinterviewworkflow">Live Interview webhooks process flow</h3>
+              <img
+                alt="Diagram of webhooks process flow for CodeSignal Live Interview"
+                width="100%"
+                src="../images/live-interview-events.svg"
+              />
+            </div>
+            <div class="subsection">
+              <h3 id="liveInterviewFinished">liveInterviewFinished</h3>
+              <p>
+                Fired when a interviewer finishes a live interview or when a live interview timeout
+              </p>
+              <div>Request format:</div>
+              <pre><code>{
+  event: 'liveInterviewFinished',
+  triggeredOn: number,
+  payload: {
+    interviewId: LiveInterviewId;
+    candidateName: Maybe<string>;
+    candidateEmail: Maybe<string>;
+    startDate: Maybe<number>;
+    endDate: Maybe<number>;
+    questionsSolved: number;
+    questionsAttempted: number;
+    url: string;
+  }
+};</code></pre>
+              <div>Sample request:</div>
+              <pre><code>{
+  event: 'liveInterviewFinished',
+  triggeredOn: 1553720789347,
+  payload: {
+    interviewId: 'zWve8sgiLJit6isiM',
+    candidateName: 'Jane Doe',
+    candidateEmail: 'jane.doe@gmail.com',
+    startDate: 1727896812565,
+    endDate: 1727896849868,
+    questionsSolved: 1,
+    questionsAttempted: 2,
+    url: https://app.codesignal.com/client-dashboard/interviews?status=over&liveInterviewId=zWve8sgiLJit6isiM
```

Comment:
Doesn't have to be a part of the example, but URLs we return in the webhook must contain access token so that even unauthenticated users can view the result.